### ISSUE TITLE
Sorry, 'x' is not a valid project name. A project with this name already exists

### ISSUE THREAD

**Issue by Pithikos** [pulumi/pulumi#12209]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

Following the current documentation a lot of new users end up not being able to start a new project.

```
> mkdir myproject && cd myproject
> pulumi new kubernetes-gcp-python
# give name myproject when asked for name
Sorry, 'myproject' is not a valid project name. A project with this name already exists
```

What I've done:
1. Delete project from the web UI
2. Reinstall pulumi cli

Still same issue and I have no idea how to reset/delete this.

### Affected area/feature

Starting a new project. Documentation is lacking for beginners.

**Comment by susanev** [comment]
@Pithikos do you have multiple pulumi accounts? or do you use multiple orgs?

**Comment by Pithikos** [comment]
@susanev ok, I figured running `rm /home/$USER/.pulumi` and re-installing worked.

Yes, I initially used a different email. And I just saw that I could probably use `pulumi whoami`. But it would be great if someone new just trying out Pulumi doesn't need to spend so much time diging around getting headaches. The error message could easily provide more information and next steps to resolve

**Comment by susanev** [comment]
@Pithikos im glad you figured it out! there is also `pulumi about`.

but im a little bit confused. you should not have gotten this error if you did not already have a project named myproject. are you saying you hit this the very first time you already had a project named myproject? or that you thought you deleted that project, but it wasn't actually gone yet so you were expecting to be able to reuse the name?

**Comment by Pithikos** [comment]
I used the wrong email the first time setting this up. Then I made a different account and went on with the same CLI. But it wasn't obvious at all that pulumi CLI was still connected to the previous account. Re-installing the CLI gave the same issue since the configuration probably was still the old one.

I think the CLI needs better commands. E.g. I still need to go to the web many many times to delete the same project when trying out templates. Preeeeeeetty damn annoying for a tool supposed to automate things. My whining is over.
